llvm/docs/CommandGuide/llvm-objcopy.rst

```diff
@@ -450,6 +450,15 @@ them.
  Set the start address of the output to ``<addr>``. Overrides any previously
  specified :option:`--change-start` or :option:`--adjust-start` options.
 
+.. option:: --set-symbol-visibility <symbol>=<visibility>
+
+ Change the visibility of a symbol to the specified value.
+
+.. option:: --set-symbols-visibility <filename>=<visibility>
+
+ Reads a list of symbols from <filename> and changes their visibility to the
```

Comment:
Imperative sentence like other options?

Reply:
I replied in another comment:
>I copied this description from other options, localize- globalize- strip- weaken- keep- symbols all use this wording. Should I change it?

Reply:
Yes, there are ~8 options that are inconsistent with others. Actually there is a larger problem that many help messages have not be synced with the clearer semantics: llvm/docs/CommandGuide/llvm-objcopy.rst

I am working on a patch to fix these help messages.

For the new options, definitely use imperative sentences as llvm/docs/CommandGuide/llvm-objcopy.rst uses.

Reply:
#82830